### ISSUE TITLE
Use limb enums in limb structs

### ISSUE THREAD

**Issue by Dragorn421** [NEstelami/ZAPD#206]
Now:
```c
StandardLimb object_daiku_Limb_007900 = { 
    { 1950, 0, 0 }, 0x0F, 0xFF,
    object_daiku_DL_005CF0
};
```

`0xFF` is known as `LIMB_DONE` [in oot `z64animation.h`](https://github.com/zeldaret/oot/blob/dcb8357f72e8a2f0b37b2b2cd6f36933194b32bd/include/z64animation.h#L15)

More complicated, but eventually we probably want to also use enums for all limb indices. Ideally use user-made ones.
I don't think there's much that can be done on that front before we move to commiting assets headers onto the repo

Ideally:
```c
StandardLimb object_daiku_Limb_007900 = { 
    { 1950, 0, 0 }, LIMB_OBJECT_DAIKU_0F, LIMB_DONE,
    object_daiku_DL_005CF0
};
```

**Comment by AngheloAlf** [comment]
I have been thinking in this idea for a long time. ZAPD producing the enums for the limbs make all the sense to me
What do you think about something like this for user-made limb indices?
```xml
<Limb Name="gZoraRightFootLimb" EnumName="ZO_LIMB_RIGHT_FOOT" LimbType="Standard" Offset="0xBEC0"/>
```

This could generate the limb enum in the header automatically. A placeholder name could be used for limbs which doesn't provide a `EnumName`

**Comment by NEstelami** [comment]
Addressed in PR #221.
